models/types/ownertype.go

```diff
@@ -0,0 +1,26 @@
+package types
```

Comment:
About the file path and the package name.
We have many `XxxType` in different files now, if we add `OwnerType` into `types` package in this PR, does this mean we should move the others into this folder/package in the future?

Reply:
I think so, if a `type` is shared by different packages.

Reply:
I see. How about use `shared_types` to make it more clearly to other maintainers?

Reply:
maybe `sharedtypes`? I guess golang doesn't like underline in package name ...

Reply:
Ah, I edited my comment.
Maybe `models/shared/ownertype.go` and package name `types` is good.